docs/index.bs

```diff
@@ -2975,6 +2975,30 @@ spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
           1. Set |serviceWorker|'s [=start status=] to null.
   </section>
 
+  <section>
+    <h3 id="terminate-service-worker-fetch-algorithm">Terminating a Service-Worker Fetch</h3>
+
+    A <dfn export>service worker fetch controller</dfn> is a [=/struct=]. It has the following [=struct/items=]:
+
+    <section dfn-for="service worker fetch controller">
+        : <dfn>state</dfn> (default "<code>ongoing</code>")
+        :: "<code>ongoing</code>" or "<code>terminated</code>".
+
+        : <dfn>preload fetch controller</dfn> (default null)
```

Comment:
I'm not sure we want to reuse the word preload for this, but I'm also not sure the original controller needs to be installed here. Can it not forward it being aborted?

Reply:
It's used to forward being aborted... I can rename but I don't entirely understand the comment

Reply:
It's used to forward being aborted... I can rename but I don't entirely understand the comment